packages/back-end/src/models/FeatureModel.ts

```diff
@@ -88,7 +88,7 @@ export async function createFeature(data: FeatureInterface) {
 }
 
 export async function deleteFeature(organization: string, id: string) {
-  await FeatureModel.deleteOne({ organization, id });
+  return await FeatureModel.deleteOne({ organization, id });
```

Comment:
If this fails to delete, we should probably just throw an error instead of returning the Mongo response.